README.md

```diff
@@ -71,7 +77,9 @@ Belle is friends with Mrs. Potts
 
 Write your code here:
 ```ruby
-# code here
+friends.each { |name|
+    print "Belle is friends with #{name}"
+}
```

Comment:
:+1:

(Although when we use the `{ }` enumerable syntax, we keep it all on one line.)